Sources/Entities/Fish.es

```diff
@@ -268,7 +268,7 @@ procedures:
     // declare yourself as a model
     InitAsModel();
     // fish must not go upstairs, or it will get out of water
-    SetPhysicsFlags((EPF_MODEL_WALKING|EPF_HASGILLS)&~EPF_ONBLOCK_CLIMBORSLIDE|EPF_ONBLOCK_SLIDE);
+    SetPhysicsFlags(((EPF_MODEL_WALKING|EPF_HASGILLS)&~EPF_ONBLOCK_CLIMBORSLIDE)|EPF_ONBLOCK_SLIDE);
```

Comment:
Ok

It's awesome that you look at what the community is doing with your code and clarify issues like this, thanks a lot!